privacyidea/lib/crypto.py

```diff
@@ -613,81 +630,125 @@ def zerome(bufferObject):
     return
 
 
+def _slow_rsa_verify_raw(key, sig, msg):
+    assert isinstance(sig, six.integer_types)
+    assert isinstance(msg, six.integer_types)
+    if hasattr(key, 'private_numbers'):
+        pn = key.private_numbers().public_numbers
+    elif hasattr(key, 'public_numbers'):
+        pn = key.public_numbers()
+    else:
+        raise TypeError('No public key')
+
+    # compute m**d (mod n)
+    return msg == pow(sig, pn.e, pn.n)
```

Comment:
Hm, would it be possible to use https://cryptography.io/en/latest/hazmat/primitives/asymmetric/rsa/#cryptography.hazmat.primitives.asymmetric.rsa.RSAPublicKey.verify to verify legacy signatures?

Reply:
As it turns out, cryptography doesn't offer the validation of unpadded "textbook" RSA by design. Resolving this.